loyalty_partner_applicability/__manifest__.py

```diff
@@ -2,7 +2,8 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Loyalty Partner Applicability",
-    "summary": "Enables the definition of a customer filter for promotion rules that will "
+    "summary": "Enables the definition of a customer filter for promotion "
+    "rules that will "
     "only be applied to customers who meet the specified conditions in the filter.",
     "version": "16.0.2.0.2",
```

Comment:
The version must be 17

Reply:
Is modified in the following commit: [[MIG] loyalty_partner_applicability: Migration to 17.0](https://github.com/OCA/sale-promotion/pull/225/commits/389f92b663575bcd4f2cee2c64ac22edf0bb6b26)

Reply:
Right, I was confused about commit, sorry

Can you push again to be able to run runbot? 

Thanks!